apiary-metastore-events/kafka-metastore-events/pom.xml

```diff
@@ -20,55 +20,18 @@
     <kafka.version>2.3.1</kafka.version>
   </properties>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
-      <version>${slf4j.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hive</groupId>
-      <artifactId>hive-common</artifactId>
-      <version>${hive.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hive</groupId>
-      <artifactId>hive-metastore</artifactId>
-      <version>${hive.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hive</groupId>
-      <artifactId>hive-exec</artifactId>
-      <version>${hive.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-common</artifactId>
-      <version>${hadoop.version}</version>
-      <scope>provided</scope>
-    </dependency>
-
-    <!--test-->
-    <dependency>
-      <groupId>org.assertj</groupId>
-      <artifactId>assertj-core</artifactId>
-      <version>${assertj.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-  </dependencies>
-
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.expediagroup.apiary</groupId>
+        <artifactId>apiary-hive-events</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.kafka</groupId>
+        <artifactId>kafka-clients</artifactId>
+        <version>${kafka.version}</version>
```

Comment:
Do you want to add the `test` scope here or have it downstream? I don't mind but we should be consistent in how we do it as much as possible across the whole project.

Reply:
I don't have too strong of an opinion and could go either way. A lot of this update was fighting maven to get all the deps to behave correctly 🙃.

When I tried to add the test scoped packages (i.e junit / mockito / etc) to the dep management here, I started getting conflicts with the deps pulled from the root parent pom. So I went w/ the route of having the downstream poms pulling directly from the parent and controlling the versions that way

Reply:
OK, so let's go with having the scope declared consistently in the child project not the parent dependency management section then (unless someone massively disagrees). I remember reading an article once about some subtle differences and that was the recommended approach.